app/controlplane/pkg/policies/policyprovider.go

```diff
@@ -53,10 +54,44 @@ func (p *PolicyProvider) Resolve(policyName string, token string) (*schemaapi.Po
 
 	// the policy name might include a digest in the form of <name>@sha256:<digest>
 	policyName, digest := policies.ExtractDigest(policyName)
+
+	var policy schemaapi.Policy
+	// old, deprecated
+	endpoint := fmt.Sprintf("%s/%s", p.host, policyName)
+	if p.uri != "" {
+		endpoint = fmt.Sprintf("%s/policies/%s", p.uri, policyName)
```

Comment:
so this is breaking compatibility or requiring the previous configuration?

wouldn't make more sense if we only have `host` in the provider struct but it's set either by the new URI object or by extracting the URI from the host?

Reply:
I've just did it this way. If no URL is provided, it will do the best effort and extract the URL from there. If the URL is provided, it will use it.
Added a bunch of tests to ensure this behaviour is correct.